### PR TITLE
Drop Vault requirement

### PR DESCRIPTION
**Pull request by fromonesrc**
This may be an overly naïve approach but spiking this to start conversation.

### CODE REVIEW

File: consul-exports.ctmpl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+{{scratch.Set "env" (env "APP_ENV") -}}
+{{$appname := env "APP_NAME" -}}
```
Comment (d4v3y0rk) [inline consul-exports.ctmpl]: you might need this code in the vault ctmpl file as well.

File: entrypoint.sh [hunk 2]
```diff
@@ -7,16 +7,31 @@ then
...
+  then
+    source /tmp/vault-exports.sh
+  else
+    echo "======== Vault may be misbehaving. If you are seeing this in prod, Engineering Ops have been alerted. ========"
+    exit 1
+  fi
+else
+  echo "VAULT_TOKEN or VAULT_ADDR are not set, skipping Vault exports"
 fi
 exec "$@"
```
Comment (tecnobrat) [inline entrypoint.sh]: You've got `exec` lines in here twice, should only be in here once.

**Comment by d4v3y0rk** [comment]
the block of code in entrypoint.sh that actually runs the consul-template command will have to be duplicated also to run each `.ctmpl` file based on the presence of the respective env variables.

File: entrypoint.sh [hunk 3]
```diff
@@ -7,16 +7,31 @@ then
...
+  echo "CONSUL_ADDR are not set skipping Consul exports"
+fi
+exec "$@"
+
+
+if [ ${VAULT_TOKEN} ] && [ ${CONSUL_ADDR} ] && [ ${VAULT_ADDR} ] 
+then
+  if consul-template -consul=$CONSUL_ADDR -template=/vault-exports.ctmpl:/tmp/vault-exports.sh -once -max-stale=0
+  then
+    source /tmp/vault-exports.sh
```
Comment (d4v3y0rk) [inline entrypoint.sh]: I think we will need to write two exports.sh files and source them both as well.

File: consul-exports.ctmpl (hunk 1 above)
Comment (fromonesrc) [inline consul-exports.ctmpl]: @d4v3y0rk it's [already there](https://github.com/articulate/docker-consul-template-bootstrap/blob/drop-vault-req/export-vault.ctmpl#L3). Am I misunderstanding?

**Comment by fromonesrc** [comment]
I think I've addressed all the comments.

File: consul-exports.ctmpl (hunk 1 above)
Comment (d4v3y0rk) [inline consul-exports.ctmpl]: yep, you are correct sir

**Comment by plukevdh** [comment]
Fixed for changes with the update to consul-template 0.8.1 and bumped to the real release version

**Comment by plukevdh** [comment]
@fromonesrc i can confirm this works locally

**Comment by fromonesrc** [comment]
Thanks @plukevdh!

**Pull request merged**